modules/RTC/BandwidthLimiter.js

```diff
@@ -0,0 +1,80 @@
+/* global __filename */
+
+import transform from 'sdp-transform';
```

Comment:
can we have the imports sorted alphabetically ? we try to do that in jitsi-meet

Reply:
fixed